docs/react-native-devtools.md

```diff
@@ -0,0 +1,99 @@
+---
+id: react-native-devtools
+title: React Native DevTools
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native DevTools is our new debugging experience featuring an end-to-end rewrite of our debugger stack. It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
+
+![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
+
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+
+<details>
+<summary>**💡 Compatibility** — released in 0.76</summary>
+
+React Native DevTools supports all React Native 0.76 apps (and onwards) running Hermes. It replaces the previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points.
+
+It is not possible to set up React Native DevTools with any older versions of React Native.
```

Comment:
maybe we can add a note here, if we want to be explicit.

```suggestion
:::note
React Native DevTools has been released in React Native 0.76.
It is not possible to set up React Native DevTools with any older versions of React Native.
:::
```

Reply:
This is already in an expandable `<details>` block; this also allows the wording to be a bit more extended and specific.